src/coreclr/src/nativeaot/System.Private.CoreLib/src/System/InvokeUtils.cs

```diff
@@ -587,16 +590,12 @@ internal static object DynamicInvokeCallTemplate(object thisPtr, IntPtr methodTo
             return null;
         }
 
-        [DebuggerStepThrough]
```

Comment:
I was thinking that we would want to add support for this (and DebuggerGuidedStepThrough) at some point so that we can have a "nice" stepping behavior. I think it can be done with the F00F00 and FEEFEE line numbers. It's Windows-specific, but if the platform allows that, it's nice to use it.

Reply:
Agree. The ones I have deleted here seems to be leftovers from when the code was factored differently. E.g.:

CallDynamicInvokeMethod (marked with `DebuggerGuidedStepThroughAttribute`)
-> `DynamicInvokeParamHelperCore` (no marked)
-> `DynamicInvokeBoxedValuetypeReturn` (no marked)
-> `DynamicInvokeUnboxIntoActualNullable` (I have deleted `DebuggerStepThrough` on it in this PR)